src/assets/js/redirect.js

```diff
@@ -0,0 +1,12 @@
+var fullPath = location.pathname + location.hash
+
+fetch('/redirects.json')
+  .then((response) => response.json())
+  .then((json) => {
+  var redirectArray = json.redirects
+  var redirectResult = redirectArray.find(item => item.from === fullPath);
+
+  if (redirectResult) {
+    location = redirectResult.to
+  }
+});
```

Comment:
Since #984 we've adopted [Standard](https://standardjs.com) on this repo. It's not yet integrated with the CI (see https://github.com/dxw/playbook/issues/1031), so non-conformant code won't produce errors, but it would be good to keep new JS code in line if possible. Using an auto-formatting extension in VSCode, it fixes this to the following to align with Standard:

```
const fullPath = location.pathname + location.hash

fetch('/redirects.json')
  .then((response) => response.json())
  .then((json) => {
    const redirectArray = json.redirects
    const redirectResult = redirectArray.find(item => item.from === fullPath)

    if (redirectResult) {
      location = redirectResult.to
    }
  })
```

Would you be able to update the code to match?

Reply:
Yeah no problem. I'll sort that now

Reply:
In reference to the points above:

1. I will add some details to the CONTRIBUTING.md file.
2. I think as long as it's documented we should keep both for now and only use the redirect manager if required. We could change this down the line if we think it's worth it.
3. I'll add an extra ticket. We could probably put a simple bit of JS in there to hide the 404 container until the redirect script is complete
4. That would be a good feature. I'll add it to the issues for now